### PR TITLE
Fix for Opencart 2.2.0.0

### PR DESCRIPTION
**Pull request by maikeldoren**
This is a fix i made for opencart 2.2.0.0.

### CODE REVIEW

File: admin/controller/payment/mollie/base.php [hunk 1]
```diff
@@ -478,26 +478,57 @@ private function validate ($store = 0)
...
-			DIR_CATALOG . "view/theme/default/template/payment/mollie_checkout_form.tpl",
-			DIR_CATALOG . "view/theme/default/template/payment/mollie_return.tpl",
-			DIR_CATALOG . "view/theme/default/template/payment/mollie_return_2.tpl",
-		);
+		
+			if ($this->isOpencart22())
+			{
+			$mod_files  = array(
+				DIR_APPLICATION . "controller/payment/mollie/base.php",
+				DIR_APPLICATION . "language/en-gb/payment/mollie.php",
```
Comment (RicardoJ) [inline admin/controller/payment/mollie/base.php]: Hello Maikel,

Thank you for your contribution to this module. Is it true that only the language follows a different structure in 2.2?

**Comment by maikeldoren** [comment]
Hello RicardoJ,

No not only the language has a different structure, the template has also changed.
but because of the nice code in Mollie ($possible_paths[]) and my changes ($default_path in isOpencart22()) it already works.

But some how you still need to change the "english" folder to "en-gb" and the "dutch" folder to "nl-nl" etc.

**Comment by RicardoJ** [comment]
Thank you for your brief explanation. I'll commit some extra changes against your PR and then release a new version of the module.

**Comment by RicardoJ** [comment]
I am using

**Comment by maikeldoren** [comment]
No problem happy to help.

**Comment by RicardoJ** [comment]
Your changes have been released in 6.3.0.

**Pull request closed without merging**